README.md

```diff
@@ -109,7 +109,9 @@ Check the full reference here: https://dandelion.eu/docs/api/datatxt/li/v1/
 var dandelion = require("node-dandelion");
 dandelion.configure({
   "app_key":"YOUR_APP_KEY",
-  "app_id":"YOUR_APP_ID"
+  "app_id":"YOUR_APP_ID",
```

Comment:
From what I gathered, they don't allow using the app_id and app_key at all. 

I would remove these two parameters as well as all the references to them in the code.

I don't have a lot of free time unfortunately but since this is my repo I'll try to update it as soon as possible if you don't have enough time to do that.